app/models/early_career_payments/eligibility.rb

```diff
@@ -174,6 +174,138 @@ def award_amounts
       award_amounts
     end
 
+    AwardAmount = Struct.new(
+      :itt_subject,
+      :itt_academic_year,
+      :claim_academic_year,
+      :base_amount,
+      :uplift_amount
+    )
+
+    def first_eligible_itt_academic_year
+      award_amounts = [
+        AwardAmount.new(
+          :mathematics,
+          AcademicYear.new(2018),
+          AcademicYear.new(2021),
+          5_000,
+          7_500
+        ),
+        AwardAmount.new(
```

Comment:
@nomasprime You may remember that back in the following PR https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/1403 I had introduced a `yml file` to hold all this data. I think I prefer it in that format over this, as it is easier to read (IMHO), and can easily be updated and then loaded through something like the following:

```ruby
MATRIX = YAML.safe_load(File.read(File.expand_path("../eligibility_matrix.yml", __FILE__))).freeze
```

Reply:
Think we can achieve similar legibility without overhead of loading from YAML.